DiceKitTests/ProbabilityMass_Tests.swift

```diff
@@ -258,4 +259,31 @@ extension ProbabilityMass_Tests {
         expect(probMass) == expectedProbMass
     }
     
+    func test_maxOutcomeOperator() {
```

Comment:
These tests are in the wrong section. You are testing the operation, so they should be `test_maximumOutcome_shouldReturnMaximumOutcomeWithOutcomes()`/`test_maximumOutcome_shouldReturnMaNilWithoutOutcomes()` in the operation section.

This section is for any operators (like `*`) that you could have (we won't have any for this).